src/server/game/Entities/Player/Player.cpp

```diff
@@ -4237,11 +4239,24 @@ void Player::DeleteFromDB(ObjectGuid::LowType lowGuid, uint32 accountId, bool up
             }
         default:
             LOG_ERROR("entities.player", "Player::DeleteFromDB: Unsupported delete method: {}.", charDelete_method);
+            deleted = false;
             return;
     }
 
+    if (deleted)
+    {
+        const CharacterCacheEntry* cache = sCharacterCache->GetCharacterCacheByGuid(playerGuid);
```

Comment:
```suggestion
        CharacterCacheEntry const* cache = sCharacterCache->GetCharacterCacheByGuid(playerGuid);
```

Reply:
From what I read there is no difference between the two, is this the preferred codestyle for AC?

Reply:
Yes

Reply:
You can find more on our codestyle here https://www.azerothcore.org/wiki/cpp-code-standards

Reply:
Ah cheers, I looked for that page but couldn't find it 😄